torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -81,6 +81,18 @@ def only_consist_of(var, types):
     return False
 
 
+# A more read-able syntax sugar for creating a UserFunctionVariable for f
+# and run call_function on it. Make it return a function to preserve the calling
+# convention of the original f.
+def _make_inlined(tx, f):
+    assert isinstance(f, Callable)
```

Comment:
Do you care if it is instance of Callable or `callable(f)` evaluate to true?

Reply:
yeah, you're right, callable(f) would be more accurate.